SPECS/protobuf/protobuf.spec

```
@@ -13,6 +13,8 @@ BuildRequires:  libstdc++
 BuildRequires:  cmake
 BuildRequires:  unzip
 BuildRequires:  abseil-cpp-devel
+BuildRequires:  gtest-devel
+BuildRequires:  gmock-devel
```

To speed up non-test builds and reduce dependencies, please surround test-only deps with `with_check`:
```suggestion
%if 0%{?with_check}
BuildRequires:  gtest-devel
BuildRequires:  gmock-devel
%endif
```